source/php/Component/Acceptance/acceptance.json

```diff
@@ -19,6 +20,18 @@
        "host" : "host ex. youtube.com",
        "name" : "name of the host"
     },
+    "types": {
+        "labels": "array|boolean",
```

Comment:
The description of "labels" says JSON string which means it is a string. See how its used in the code to determine if its a string or an array|object